File: appambit-testapp/src/main/java/com/appambit/testapp/utils/storage/DataStoreTest.java
package com.appambit.testapp.utils.storage;
import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import com.appambit.sdk.AppConstants;

public class DataStoreTest extends SQLiteOpenHelper {
    public DataStoreTest(Context context) {
        super(context, AppConstants.DATABASE_NAME, null, AppConstants.DB_VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase sqLiteDatabase) {}

    @Override
    public void onUpgrade(SQLiteDatabase sqLiteDatabase, int i, int i1) {}
}

File: samples/kotlin-app/src/main/java/com/appambit/kotlinapp/utils/storage/DataStoreTest.kt
package com.appambit.kotlinapp.utils.storage

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper
import com.appambit.sdk.AppConstants

class DataStoreTest(context: Context?) :
    SQLiteOpenHelper(context, AppConstants.DATABASE_NAME, null, AppConstants.DB_VERSION) {
    override fun onCreate(sqLiteDatabase: SQLiteDatabase?) {}

    override fun onUpgrade(sqLiteDatabase: SQLiteDatabase?, i: Int, i1: Int) {}
}